README.md

```diff
@@ -1,123 +1,69 @@
-# NXT, standalone part
+# Dawn, a WebGPU implementation
+
+Dawn (formerly NXT) is a open-source and cross-platform implementation of the work-in-progress WebGPU standard.
+It exposes a C/C++ API that maps almost one-to-one to the WebGPU IDL and can be managed as part of a larger system such as a Web browser.
+
+Dawn provides several WebGPU building blocks:
+ - **WebGPU C/C++ headers** that applications and other building blocks use.
+ - **A "native" implementation of WebGPU** using platforms' GPU APIs:
+  - **D3D12** on Windows 10
+  - **Metal** on OSX (and eventually iOS)
+  - **Vulkan** on Windows, Linux (eventually ChromeOS and Android too)
+  - OpenGL as best effort where available
+ - **A client-server implementation of WebGPU** for applications that are in a sandbox without access to native drivers
+
+## Directory structure
+
+- `dawn.json`: the description of the API used to drive code generators.
+- `examples`: some examples showing how Dawn is used.
+- `generator`: code generator for files produces from `dawn.json`
+  - `templates`: Jinja2 templates for the generator
+- `scripts`: script to support things like continuous testing, build files, etc.
+- `src`: 
+  - `common`: helper code shared between core Dawn libraries and tests/samples
+  - `dawn_native`: native implementation of WebGPU, one subfolder per backend
+  - `dawn_wire`: client-server implementation of WebGPU
+  - `include`: public headers for Dawn
+  - `tests`: internal Dawn tests
+    - `end2end`: WebGPU tests performing GPU operations
+    - `unittests`: unittests and by extension tests not using the GPU
+      - `validation`: WebGPU validation tests not using the GPU (frontend tests)
+  - `utils`: helper code to use Dawn used by tests and samples
+- `third_party`: where dependencies are, and their buildfiles.
```

Comment:
"where depencies are" seems grammatically awkward, but I can't come up with anything better that fits on one line. :(

Reply:
Went with

> directory where dependencies live as well as their buildfiles.